keentools/geotracker/interface/panels.py

```diff
@@ -652,6 +652,24 @@ def _tracking_remove_keys_row(self, settings: Any, layout: Any) -> None:
         btn.operator(GTConfig.gt_clear_tracking_menu_exec_idname,
                      text='', icon='X')
 
+    def _tracking_locks(self, settings: Any, layout: Any, geotracker: Any) -> None:
```

Comment:
Incorrect indices of locks. Should be the following:
translation x - 3
translation y - 4
translation z - 5
rotation x - 0
rotation y - 1
rotation z - 2

Reply:
Fixed